src/hooks/use-ast.ts

```diff
@@ -27,6 +27,7 @@ export function useAST() {
 		case "javascript": {
 			try {
 				const ast = espree.parse(code.javascript, {
+					// @ts-expect-error
```

Comment:
To the team: 

The `@ts-expect-error` is used due to a mismatch between the latest versions of `espree` and `@types/espree`.

I think we can revisit this or wait until `@types/espree` is updated to the latest version.

Alternatively, adding types directly to `espree` ([https://github.com/eslint/js/issues/529](https://github.com/eslint/js/issues/529)) or submitting a PR to [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) could also make sense.

I’d like to ask @eslint/eslint-team what approach we should take here.

Reply:
Updating `@types/espree` will take a long time -- we went through this with `@types/eslint` and it just wasn't worth the effort. I'd say just suppress the error.

Reply:
@nzakas Sounds good to me 👍 

---

```suggestion
					// @ts-expect-error mismatch between the latest release of `espree` and `@types/espree`.
```

@xbinaryx Can we add a brief description explaining why this error occurs? 😄